dll/cpl/appwiz/createlink.c

```diff
@@ -450,7 +450,14 @@ ShowCreateShortcutWizard(HWND hwndCPl, LPWSTR szPath)
         StringCchCopyW(pContext->szOldFile, _countof(pContext->szOldFile), szPath);
         pch = PathFindFileNameW(pContext->szOrigin);
         if (pch && *pch)
+        {
+            /* build szDescription */
+            StringCchCopyW(pContext->szDescription, _countof(pContext->szDescription), pch);
+            *pch = 0;
```

Comment:
Not sure whether this is needed.

Reply:
The result of `szOrigin` is expected as a folder location. `*pch = 0;` will cut the file title part.